docs/HowTo/Configure/FreeGas.md

```diff
@@ -1,54 +1,60 @@
-description: Configuring Free Gas Networks 
+description: Configuring free gas networks
 <!--- END of page meta data -->
 
-# Free Gas Networks 
+# Free gas networks
 
-Transactions use computational resources so have an associated cost. Gas is the cost unit and the gas 
-price is the price per gas unit. The transaction cost is the gas used * gas price. 
+Transactions use computational resources so have an associated cost. Gas is the cost unit and the
+gas price is the price per gas unit. The transaction cost is the gas used * gas price.
 
-In public networks, the transaction cost is paid in Ether by the account submitting the transaction.
-The transaction cost is paid to the miner (or validator in PoA networks) that includes the transaction in a block.  
+In public networks, the account submitting the transaction pays the transaction cost, in Ether.
+The miner (or validator in PoA networks) that includes the transaction in a block receives
+transaction cost.
 
-In many private networks, the validators are run by the network participants and do not require gas as an 
-incentive to participate.  Generally, networks that do not require gas as an incentive, configure the gas price to be 0 (that is, make gas free). 
-Some private networks may allocate Ether and use a non-zero gas price to limit resource use.  
+In many private networks, network participants run the validators and do not require gas as an
+incentive. Generally, networks that do not require gas as an incentive, configure the gas price to
```

Comment:
Feels like a word is missing in this sentence? 

Generally, in networks ....

Reply:
I agree the sentence is a bit poorly worded. I changed to:
"Networks that do not require gas as an incentive usually configure the gas price to
be zero (that is, make the gas free)."